package.json

```diff
@@ -76,7 +76,7 @@
   },
   "dependencies": {
     "@dotlottie/react-player": "^1.6.3",
-    "@expensify/react-native-live-markdown": "0.1.210",
+    "@expensify/react-native-live-markdown": "0.1.215",
```

Comment:
@mananjadhav Please make sure to build and test changes with the hybrid app setup, this change is causing hybrid app build to fail as there is a patch that has to be updated https://github.com/Expensify/App/actions/runs/12720655080/job/35462635073